src/src.builds

```diff
@@ -12,7 +12,7 @@
 
   <PropertyGroup>
     <!-- TODO: We should see about generating this from scratch instead of relying on a previous deps file as a template -->
-    <_ToolsDotNetCliSharedFxPath>$(ToolsDir)dotnetcli\shared\Microsoft.NETCore.App\1.1.0</_ToolsDotNetCliSharedFxPath>
+    <_ToolsDotNetCliSharedFxPath>$(ToolsDir)dotnetcli\shared\Microsoft.NETCore.App\1.1.1</_ToolsDotNetCliSharedFxPath>
```

Comment:
Oops, I believe it should be. This is another annoying place where versions are hardcoded...